front/src/modules/trainschedule/components/ImportTrainSchedule/ImportTrainScheduleTrainDetail.tsx

```diff
@@ -78,7 +78,7 @@ export default function ImportTrainScheduleTrainDetail({
         </span>
         <span
           className={`import-train-schedule-traindetail-stepnb ${
-            trainData.steps.length <= 2 ? 'bg-secondary' : 'bg-primary'
+            trainData.steps.length <= 2 ? 'bg-danger' : 'bg-primary'
```

Comment:
Maybe we should use a variable in our color variables? like `error`?

Reply:
Ah yes you're right, I thought danger and error60 were the same but they are not. Can you use error60 (you can see it in our `styles/_variables.scss`

Reply:
that's better, thanks !